src/components/TeacherComponents/OnboardingForm/onboardingActions/getSubscriptionStatus.ts

```diff
@@ -1,8 +1,7 @@
 import OakError from "@/errors/OakError";
-import { subscriptionResponseSchema } from "@/pages/api/hubspot/subscription";
```

Comment:
This import was causing server-side code to be pulled onto the client. I only picked this up when I tried to import the Clerk server module into the subscription API route and the built in guard against running on the client started to blow up. I originally thought this was because the Clerk server module wouldn't work in a Pages router API endpoint (which was why I went and moved everything into App router land 😭 ) but that wasn't the case.